src/lib/structures/ServerLog.js

```diff
@@ -0,0 +1,95 @@
+const { MessageEmbed } = require("discord.js");
+
+module.exports = class ServerLog {
+
+    constructor(guild) {
+        this.guild = guild;
+        this.client = guild.client;
+
+        this.type = null;
+        this.user = null;
+        this.data = null;
+    }
+
+    /**
+     * Gives a type to the server log
+     * @param {string} type type of case
+     * @returns {ServerLog}
+     */
+    setType(type) {
+        this.type = type;
+        return this;
+    }
+
+    /**
+     * Gives a type to the server log
+     * @param {Object} data type of case
+     * @returns {ServerLog}
+     */
+    setData(data) {
```

Comment:
this class is pointless if you're using one function to set all data, acts nothing like MessageEmbed

Reply:
It's still a WIP, this class has not been finished yet.

Reply:
its a review, doesnt matter if its WIP or not.